extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/DebeziumCdcSources.java

```diff
@@ -64,6 +66,24 @@ public static Builder<ChangeRecord> debezium(@Nonnull String name, @Nonnull Stri
                 (properties, eventTimePolicy) -> new ChangeRecordCdcSourceP(properties, eventTimePolicy));
     }
 
+    /**
+     * Creates a CDC source that streams change data from a
+     * Debezium-supported database to a Hazelcast Jet pipeline.
+     *
+     * @param name the name of this source, must unique, will be passed
+     *             to the underlying Kafka Connect source
+     * @param connectorClass class of the Debezium Connector which will be used for the CDC
+     * @return a builder you can use to set the source's properties and
+     * then construct it
+     */
+    @Nonnull
+    public static Builder<ChangeRecord> debezium(
+            @Nonnull String name,
+            @Nonnull Class<? extends SourceConnector> connectorClass) {
```

Comment:
We can provide Class<?> but not `extends SourceConnector` - leaking the kafka connect API to the interface.